### ISSUE TITLE
Add submenu to About OASIS+ menu in main nav

### ISSUE THREAD

**Issue by klohman** [GSA/oasis-plus#168]
In main navigation, the About OASIS+ menu item should include a submenu dropdown. 

![Image](https://github.com/GSA/oasis-plus/assets/18366192/58a39ff7-7d04-4ccf-9bed-8f37ab205daa)

**Comment by mark-meyer** [comment]
@klohman I went ahead to pulled this in — it's a quick fix. If you want to give it a quick look:  https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/mmeyer/about-menu/

**Comment by klohman** [comment]
@mark-meyer Yes looks good- thanks! I don't think PSHC needs to approve this - it can go to Done once merged.
